### ISSUE TITLE
trim spaces in text labels of data nodes

### ISSUE THREAD

**Issue by egonw** [wikipathways/reactome2gpml-converter#47]
In the current GPML created some labels have redundant spaces at the end of labels, e.g. in WP3579:

    <DataNode TextLabel="PC " GraphId="cdb40" Type="Metabolite" GroupRef="bbf32">
      <Attribute Key="cellular_location" Value="endoplasmic reticulum membrane" />
      <Attribute Key="complex_id" Value="R-ALL-5684863" />
      <Attribute Key="copies_num" Value="1" />
      <Graphics CenterX="240.0" CenterY="1851.0" Width="100.0" Height="20.0" ZOrder="32768" FontSize="10" Valign="Middle" Color="0000ff" />
      <Xref Database="ChEBI" ID="CHEBI:16110" />
    </DataNode>

**Comment by DeniseSl22** [comment]
I'll check if this is still occurring in newer versions; but this could be due to curators in Reactome accidentally adding a space. And I'll try to get the trim() integrated in the pathways.

**Comment by egonw** [comment]
Yes, still the case in the current output:

```xml
  <DataNode TextLabel="Dovitinib " GraphId="ba4b1" GroupRef="d4f5d">
    <Attribute Key="cellular_location" Value="cytosol" />
    <Attribute Key="complex_id" Value="R-ALL-2045079" />
    <Attribute Key="copies_num" Value="1" />
    <Graphics CenterX="0.0" CenterY="0.0" Width="0.0" Height="0.0" ZOrder="32768" FontSize="10" Valign="Middle" />
    <Xref Database="" ID="" />
  </DataNode>
```

**Comment by DeniseSl22** [comment]
Okay, I'll add this to the converter code asap, but will be fixed then in version 67 (or is it a necessary fix for older versions?)...

**Comment by egonw** [comment]
@DeniseSl22, please make sure you see the pull request: I already fixed the code and it's there for you to review: https://github.com/wikipathways/reactome2gpml-converter/pull/64

**Comment by egonw** [comment]
No, for version 67 sounds fine. Is that the latest version in this round, or the next? You could always opt to redo the last one you have now...

**Comment by DeniseSl22** [comment]
Okay, good. 67 would be the next version, but I can do the testing on version 66. But I want consistency for the previous version uploads, since it is otherwise to difficult for me to pinpoint where things are breaking down 9if they do).
